pkgs/applications/science/misc/netlogo/default.nix

```diff
@@ -0,0 +1,43 @@
+{ jre, stdenv, fetchurl, makeWrapper }:
+
+
+stdenv.mkDerivation rec {
+  name = "NetLogo-${version}";
+  version = "6.0.4";
+
+
+  src = fetchurl {
+    url = "https://ccl.northwestern.edu/netlogo/6.0.4/NetLogo-6.0.4-64.tgz";
```

Comment:
Maybe try to use ${version} more in this url.

Reply:
Thank you! Will do!